muesli/web/static/js/jquery/jquery.tablesorter.min.js

```diff
@@ -1 +0,0 @@
-/usr/share/javascript/jquery-tablesorter/jquery.tablesorter.min.js
```

Comment:
This also should not have been removed, fixed in the newest commit.

Reply:
Hey, all the files that have been removed were symlinks. Not files containing a line with the path. I know, that this is technically similar and github presents it in this way. But you need to change this to symlinks. You might want to rebase onto muesli-project first, since these files might have been introduced in a commit you based your work on.